server/routes/api/browser.py

```diff
@@ -33,4 +35,29 @@ def triple_api(dcid):
 @bp.route('/popobs/<path:dcid>')
 def popobs_api(dcid):
     """Returns all the triples given a node dcid."""
-    return dc.get_pop_obs(dcid)
\ No newline at end of file
+    return dc.get_pop_obs(dcid)
+
+
+@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@bp.route('/propvals/<path:prop>/<path:dcid>')
+def get_property_value(dcid, prop):
+    response = fetch_data('/node/property-values', {
+        'dcids': [dcid],
+        'property': prop,
+    },
+                          compress=False,
+                          post=False)
+    result = {}
+    result["property"] = prop
+    result["values"] = response.get(dcid, {})
+    return Response(json.dumps(result), 200, mimetype='application/json')
+
+
+@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+@bp.route('/proplabels/<path:dcid>')
+def get_property_labels(dcid):
+    response = fetch_data('/node/property-labels', {'dcids': [dcid]},
```

Comment:
similar for this one, can use https://github.com/datacommonsorg/website/blob/99bc65c31ea19c1d72b8b8a1fa667941a3dedf8a/server/services/datacommons.py#L193.

Can change  that function to remove the "out" params (so it returns all results)

Reply:
done